src/operator-docs/creation/fromPromise.ts

```diff
@@ -1,6 +1,46 @@
 import { OperatorDoc } from '../operator.model';
 
 export const fromPromise: OperatorDoc = {
-  'name': 'fromPromise',
-  'operatorType': 'creation'
+  name: 'fromPromise',
+  operatorType: 'creation',
+  signature:
+    'public static fromPromise(promise: Promise<T>, scheduler: Scheduler): Observable<T>',
+  parameters: [
+    {
+      name: 'promise',
+      type: 'Promise<T>',
+      attribute: '',
+      description: 'The promise to be converted.'
+    },
+    {
+      name: 'scheduler',
+      type: 'Scheduler',
+      attribute: 'optional',
+      description:
+        'An optional IScheduler to use for scheduling the delivery of the resolved value (or the rejection).'
+    }
+  ],
+  shortDescription: {
+    description: 'Converts a Promise to an Observable.'
+  },
+  walkthrough: {
+    description: `Converts an ES2015 Promise or a Promises/A+ spec compliant Promise to an Observable.
+    If the Promise resolves with a value, the output Observable emits that resolved value as a next,
+    and then completes. If the Promise is rejected, then the output Observable emits the corresponding Error.`
+  },
+  examples: [
+    {
+      name: 'Convert the Promise returned by Fetch to an Observable',
+      code: `
+         import {fromPromise} from 'rxjs/observable/fromPromise';
+
+        const result = fromPromise(fetch('http://myserver.com/'));
+        result.subscribe(x => console.log(x), e => console.error(e));`,
```

Comment:
Please add expected output also.

Reply:
I think this would be rather confusing in this case. Because the operator is more or less like a cast and doesn't manipulate the output. What do you think? Of course I can add an output, but I think it won't add much value in this case.